### PR TITLE
Pulling a human mob with a slowdown greater than yours now puts you to their speed.

### PR DESCRIPTION
**Pull request by Hockaa**
Adds a check to see if your pull target is human, and if so, if there species slowdown is greater than yours it gets added to the movement_delay tally var.

### CODE REVIEW

**Comment by Hockaa** [comment]
!review

File: code/modules/mob/living/carbon/human/human_movement.dm [hunk 1]
```diff
@@ -4,6 +4,14 @@
 	if(species.slowdown)
 		tally = species.slowdown
 
+	if(ishuman(pulling))
+		var/mob/living/carbon/human/H = pulling
+		var/pulldelay
+		pulldelay = H.species.slowdown
+		if(pulldelay > species.slowdown)
+			tally = H.species.slowdown
```
Comment (mikomyazaki) [inline code/modules/mob/living/carbon/human/human_movement.dm:12]:
```suggestion
		if(H.species.slowdown> species.slowdown)
			tally = H.species.slowdown
```

File: code/modules/mob/living/carbon/human/human_movement.dm [hunk 2]
```diff
@@ -169,3 +176,7 @@
 
 /mob/living/carbon/human/mob_negates_gravity()
 	return (shoes && shoes.negates_gravity())
+
+/mob/living/carbon/human/proc/ClothesSlowdown()
+	for(var/obj/item/I in list(wear_suit, w_uniform, back, gloves, head))
```
Comment (mikomyazaki) [inline code/modules/mob/living/carbon/human/human_movement.dm:181]: Why only these items? Why not loop over all clothing?

Comment (Hockaa) [reply code/modules/mob/living/carbon/human/human_movement.dm:179]: That was just what's in the original loop, honestly. I didn't think too hard on it when I was writing it back into a proc.

**Pull request merged**